assessments/NWEA_Map/templates/studentAssessment.jsont

```diff
@@ -1,5 +1,5 @@
 {
-    "studentAssessmentIdentifier": "{% if TestID is not none and TestID | length%}{{TestID}}{%else%}{{ md5(generated_test_id) }}{%endif%}",
+    "studentAssessmentIdentifier": "{% if TestID is not none and TestID | length and TestID != "<NA>" %}{{TestID}}{%else%}{{ md5(generated_test_id) }}{%endif%}",
```

Comment:
Add NA filters alongside standard empty-string filters (perhaps excessive in Earthmover 0.3.x).